lang/python/python3-django-etesync-journal/Makefile

```diff
@@ -0,0 +1,39 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=django-etesync-journal
+PKG_VERSION:=1.0.2
+PKG_RELEASE:=1
+
+PYPI_NAME:=django-etesync-journal
+PKG_HASH:=716df1fc8670c79750e8a1e736b69ddbdbd238237565874ea9648387cc50626e
+
+PKG_LICENSE:=AGPL-3.0
```

Comment:
Use `AGPL-3.0-only` or `AGPL-3.0-or-later` (not sure which one applies here).

Reply:
Changed it to `AGPL-3.0-only`.